### DISCUSSION TITLE
Erro de sintaxe ao tentar implementar um Contract com flunt

### DISCUSSION THREAD

**Discussion by RafaelDifini** [balta-io/forum#235] [Tira Dúvidas] [answered]
Curso modelando dominios ricos,

Estou tendo o seguinte erro ao tentar implementar o metodo AddNotifications

![image](https://user-images.githubusercontent.com/79666617/213889549-d2344f05-927f-4efb-b510-4df1dd60e4e1.png)

vi pela documentaçao do flunt que Notifiable não se usa mais igual esta no curso e ja fiz a alteração 

tentei na implementação da classe Contract<T> porem não entendi oque posso fazer

Edit: Tentei resolver colocando o tipo do Objeto que estou trabalhando, esta correto? e tbm gostaria de saber se os metodos HasMinLen não existem mais, procurei na documentação e nao encontrei, tbm esta gerando erro.
![image](https://user-images.githubusercontent.com/79666617/213893979-74bdfc57-8fc3-4a32-a74f-83be334c978e.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @RafaelDifini, como vai?
Você está utilizando a mesma versão do Flunt que eu apresento no curso? Recomendo utilizar a mesma para seguir, depois você pode refazer o curso atualizando para versão 2 conforme exemplos do repositório (Flunt)
